### PR TITLE
my fix to get nat instance working

### PR DESCRIPTION
**Pull request by nelg**
Seems to work OK

### CODE REVIEW

**Comment by nelg** [comment]
@int128  - this works for me, and does not work without it.  Are you able to consider merging this?

**Comment by iamkhalidbashir** [comment]
Very importnat should merge

**Comment by aalimovs** [comment]
@int128 could you look at merging this please? or could you add maintainers please? thanks! 🙇

**Comment by arjitj2** [comment]
@int128 please add maintainers! This is an important PR to merge.

File: snat.sh [hunk 1]
```diff
@@ -5,13 +5,16 @@ while ! ip link show dev eth1; do
...
+
+
 # enable IP forwarding and NAT
 sysctl -q -w net.ipv4.ip_forward=1
 sysctl -q -w net.ipv4.conf.eth1.send_redirects=0
-iptables -t nat -A POSTROUTING -o eth1 -j MASQUERADE
-
-# switch the default route to eth1
-ip route del default dev eth0
+iptables -t nat -A POSTROUTING -o eth0 -j MASQUERADE
```
Comment (int128) [inline snat.sh:17]: I think this breaks the fixed source IP feature. ENI of eth0 will be changed when the instance is recreated by Auto Scaling Group.

Comment (jl-DaDar) [reply snat.sh:17]: will this be merged any time soon?

**Comment by int128** [comment]
By this pull request, the source IP address of NAT would fluctuate. I intentionally use `eth1` to pin the source IP.

I think #42 resolved the issue.

**Pull request closed without merging**